### PR TITLE
Reword UI test steps

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
Changes wording of UI gherkin test steps.

Major points:
1) for most steps, mention ``using the webUI`` when the step is supposed to use the webUI to do the thing (vs "really background setup" that can/could create users/gorups... via any means like API, occ.

2) for `Then`` steps that test something that should be observed on the UI, mention ``on the webUI`` (vs when we are happy to test if a file exists or... by doing it "underneath" with the API)

3) when a step uses words like ``browse`` or ``redirected to page`` etc, then that is a webUI-specific word, so we do not have to always explicitly say ``using the webUI``

4) In ``Given`` steps that set the scene, use "present perfect" tense - ``the user has done xyz``

5) In ``When`` steps that perform the action(s) under test, use present tense - ``the user does xyz``

6) In ``Then`` steps that test some outcome, put the word ``should`` in the sentence - ``Then the folder "sub-folder" should be listed on the webUI``

## Related Issue
https://github.com/owncloud/QA/issues/517

## Motivation and Context
Use a "standard" from of words in UI test steps so it is clear that they apply to the ``webUI`` and which steps are "setup" steps ``Given``, which are for ``When`` and which are for ``Then`` testing outcomes.

Make it like what was done to the API acceptance tests.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Test refactoring

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

Doc PR https://github.com/owncloud/documentation/issues/3829

### CODE REVIEW

File: tests/acceptance/features/bootstrap/WebUIFilesContext.php [hunk 1]
```diff
@@ -171,26 +174,27 @@ public function iAmOnTheFavoritesPage() {
...
 
 	/**
-	 * @When the files/favorites page is reloaded
+	 * @When the user reloads the current page of the webUI
+	 * @Given the user has reloaded the current page of the webUI
 	 *
 	 * @return void
 	 */
-	public function theFilesPageIsReloaded() {
+	public function theUserReloadsTheCuurentPageOfTheWebUI() {
```
Comment (patrickjahns) [inline tests/acceptance/features/bootstrap/WebUIFilesContext.php]: typo

Comment (phil-davis) [reply tests/acceptance/features/bootstrap/WebUIFilesContext.php]: fixed

**Comment by phil-davis** [comment]
I will squash after I am feeling confident :)

**Comment by phil-davis** [comment]
Backport stable10 #30765

**Comment by phil-davis** [comment]
codecov for the patch is not affected, but codecov for the project is down -17.83% - it struggles to get a useful answer after rebase-squash.

**Review by individual-it — APPROVED** [review]

**Comment by phil-davis** [comment]
Related app PRs:
- [ ] files_texteditor https://github.com/owncloud/files_texteditor/pull/228
- [ ] firewall https://github.com/owncloud/firewall/pull/468
- [ ] user_ldap

**Pull request merged**